vision/set_endpoint.rb

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def set_endpoint service_address: nil, image_path: nil
+  # [START vision_set_endpoint]
+  # service_address = "Service address, eg. 'eu-vision.googleapis.com'"
```

Comment:
I'm not sure we are allowed to call it a service_address

Reply:
why not?  the variable name is `service_address` in the client library.

Reply:
If that is the client library's variable name then it should be fine.